prac_06/car.py

```diff
@@ -0,0 +1,31 @@
+"""CP1404/CP5632 Practical - Car class example."""
+
+
+class Car:
+    """Represent a Car object."""
+
+    def __init__(self, fuel=0):
+        """Initialise a Car instance.
+
+        fuel: float, one unit of fuel drives one kilometre
+        """
+        self.fuel = fuel
+        self._odometer = 0
+
+    def add_fuel(self, amount):
+        """Add amount to the car's fuel."""
+        self.fuel += amount
+
+    def drive(self, distance):
```

Comment:
Add type hints for distance and the return value. Consider adding more details in the docstring about the return value.